fl4health/utils/random.py

```diff
@@ -60,6 +60,32 @@ def unset_all_random_seeds() -> None:
     torch.use_deterministic_algorithms(False)
 
 
+def save_random_state() -> Tuple[Tuple[Any, ...], Dict[str, Any], torch.Tensor]:
+    """
```

Comment:
I like the documentation, but it's not quite in the Google Docstring format here. That is, the return and its type aren't documented.